stacker/blueprints/rds/base.py

```diff
@@ -0,0 +1,336 @@
+from troposphere import (
+    Ref, ec2, Output, GetAtt, Not, Equals, Condition, And, Join, If
+)
+from troposphere.rds import DBInstance, DBSubnetGroup
+from troposphere.route53 import RecordSetType
+
+from ..base import Blueprint
+
+RDS_ENGINES = ["MySQL", "oracle-se1", "oracle-se", "oracle-ee", "sqlserver-ee",
+               "sqlserver-se", "sqlserver-ex", "sqlserver-web", "postgres"]
+
+# Resource name constants
+SUBNET_GROUP = "RDSSubnetGroup"
+SECURITY_GROUP = "RDSSecurityGroup"
+DBINSTANCE_NO_IOPS = "RDSDBInstanceNoIops"
+DBINSTANCE_WITH_IOPS = "RDSDBInstanceWithIops"
+DNS_RECORD = "DBInstanceDnsRecord"
+
+
+def common_parameters():
+    """ Return common parameters for all RDS stacks.
+
+    Returns:
+        dict: A dictionary of parameter definitions.
+    """
+
+    parameters = {
```

Comment:
I haven't yet figured out if that's the right thing to do here or not.  There's a lot of 'if this parameter is specified, these parameters shouldn't be' which I handled in Read Replicas by just removing the unused/disallowed ones.  Not sure what the right case should be here.  If you have any ideas, let me know.